lib/IDE/CodeCompletion.cpp

```diff
@@ -5373,13 +5379,16 @@ void CodeCompletionCallbacksImpl::doneParsing() {
   }
 
   case CompletionKind::AttributeBegin: {
-    Lookup.getAttributeDeclCompletions(IsInSil, AttTargetDK);
+    auto DK = AttTargetDK;
+    if (AttTargetIsIndependent)
+      DK = None;
```

Comment:
Why do we check this here instead of in `setAttrTargetDeclKind` ?

Reply:
No strong reason. Moved to `setAttrTargetDeclKind `.